src/rev_ai/apiclient.py

```diff
@@ -438,18 +449,19 @@ def _create_job_options_payload(
             test_mode=None,
             segments_to_transcribe=None):
         payload = {}
-        if media_url:
-            payload['media_url'] = media_url
+        if source_url:
+            payload['source_config'] = CustomerUrlData(source_url, source_auth).to_dict()
```

Comment:
My thought was it might be simpler to abstract away the structure of the request, but since those options are very much dependent on each other it might be better to show that